readme.md

```diff
@@ -94,22 +146,12 @@ puts alice.greet
 puts bob.greet
 ```
 
-<details>
-<summary>If we change `def set_name_to(some_string)` to `def christen(name)`, what else will need to change?</summary>
-`@name = some_string` must become `@name = name`, and `alice.set_name_to("Alice")` must become `alice.christen("Alice")`.
-</details>
-
-<details>
-<summary>If we change `@name = some_string` to `@username = some_string`, what else will need to change?</summary>
-Every `@name` must become `@username`.
-</details>
-
-- is `User` a(n)...
+- Is `User` a(n)...
```

Comment:
there are so many words in this concept. maybe even start w/ a glossary of terms